src/tox_uv/_venv.py

```diff
@@ -83,6 +84,21 @@ def _get_python(self, base_python: list[str]) -> PythonInfo | None:  # noqa: PLR
                     platform=sys.platform,
                     extra={},
                 )
+            if os.path.isabs(base):  # noqa: PTH117
+                return PythonInfo(
+                    implementation="CPython",
```

Comment:
Every line of the data you specified here is potentially untrue and a guess. You cannot just do a best guess. You actually need to call the interpreter and specify the correct fields.

Reply:
That is true -- however I stole this from the existing logic branch right below.

The real mismatch here is that `PythonInfo` does not allow for `None` values:
```
class PythonInfo(NamedTuple):
    implementation: str
    version_info: VersionInfo
    version: str
    is_64: bool
    platform: str
    extra: dict[str, Any]
```
in fact neither does `VersionInfo` -- not that this stopped existing code from using `None` values anyway :-)
```
class VersionInfo(NamedTuple):
    major: int
    minor: int
    micro: int
    releaselevel: str
    serial: int
```

I would be reluctant to actually call out to the interpreter for the same reason you mentioned at tox-dev/tox#3191, namely performance (and I think this would be quite fragile too)

I think the best thing to do here is to follow precedent and replace the guesses with `None` values (meaning: don't know/don't care). The one exception being `implementation` which I'll set to empty string purely because code in tox tries to `.lower()` it.

I'll amend the PR just now -- let me know if this works for you?

Reply:
<went back and re-read everything>

That is true. I stole a bunch of this from the branch immediately following. That said upon re-reading this I don't think there is a clean way of doing this because of the `PythonInfo` type expects full knowledge of everything, which in turn is made possible by `PythonSpec`. Except here we do not have specs. 

I would be reluctant to actually call out to the interpreter for the same reason you mentioned at tox-dev/tox#3191, namely performance (and I think this would be quite fragile too. Plus this is information that we would throw away anyway.)

I see three ways out of this, in ascending order of work required:

1. still create a `PythonInfo` object, but set all values to falsey values of convenience -- `""`, `0`, `False`
2. extend `PythonInfo` to accept `None` values
3. change the return type of `_get_python()` to `-> PythonInfo | str | None`

For now I will go with #1. Let me know if this sounds good

Reply:
Please don't mark comments resolved yourself. The reviewer is the one that should do that as they are the one that knows if they agree that the issue has been resolved or not. In this case i disagree with you. Below we set values from spec, not made up values. And no, those values cannot be None, because then we would not be able to guarantee that the environments respect the spec requested by the user.

Reply:
Sure, I see your point now. I was not aware that tox performed post-hoc validation on the `PythonInfo` object.

I've made the change, please re-review. I also added a test case to verify that the spec vs. interpreter mismatch definitely gets picked up as a conflict, which it does.